h/static/images/icons/mail.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="14px" viewBox="0 0 18 14" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 41.2 (35397) - http://www.bohemiancoding.com/sketch -->
+    <title>Rectangle 93 Copy 12</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <path d="M2.57142857,2 L0,2 L0,14 L18,14 L18,2 L15.4285714,2 L18,0 L0,0 L2.57142857,2 L0,2 L9,9 L18,2 L15.4285714,2 L9,7 L2.57142857,2 Z" id="Rectangle-93-Copy-12" fill="#A6A6A6"></path>
```

Comment:
Same comment as https://github.com/hypothesis/h/pull/4174/files#r91279058 for the hard-coded color here and in the other icons.